src/services/vyos-domain-resolver

```diff
@@ -119,6 +124,66 @@ def nft_valid_sets():
     except:
         return []
 
+def is_valid_ipv4_address_or_range(addr: str) -> bool:
```

Comment:
This could be a good addition to the `vyos.utils.network` module instead of local to the resolver.